src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowStateAllowedEventService.java

```diff
@@ -222,32 +235,51 @@
                 WITHDRAW_CLAIM,
                 DISCONTINUE_CLAIM,
                 CASE_PROCEEDS_IN_CASEMAN,
-                AMEND_PARTY_DETAILS
+                AMEND_PARTY_DETAILS,
+                ADD_CASE_NOTE
             )
         ),
         entry(
             CLAIM_DISMISSED_PAST_CLAIM_NOTIFICATION_DEADLINE.fullName(),
-            List.of(CASE_PROCEEDS_IN_CASEMAN)
+            List.of(
+                CASE_PROCEEDS_IN_CASEMAN,
+                ADD_CASE_NOTE
+            )
         ),
         entry(
             CLAIM_DISMISSED_PAST_CLAIM_DETAILS_NOTIFICATION_DEADLINE.fullName(),
-            List.of(CASE_PROCEEDS_IN_CASEMAN)
+            List.of(
+                CASE_PROCEEDS_IN_CASEMAN,
+                ADD_CASE_NOTE
+            )
         ),
         entry(
             PAST_APPLICANT_RESPONSE_DEADLINE_AWAITING_CAMUNDA.fullName(),
-            List.of(TAKE_CASE_OFFLINE)
+            List.of(
+                TAKE_CASE_OFFLINE,
+                ADD_CASE_NOTE
+            )
         ),
         entry(
             PAST_CLAIM_NOTIFICATION_DEADLINE_AWAITING_CAMUNDA.fullName(),
-            List.of(DISMISS_CLAIM)
+            List.of(
+                DISMISS_CLAIM,
+                ADD_CASE_NOTE
+            )
         ),
         entry(
             PAST_CLAIM_DETAILS_NOTIFICATION_DEADLINE_AWAITING_CAMUNDA.fullName(),
-            List.of(DISMISS_CLAIM)
+            List.of(
+                DISMISS_CLAIM,
+                ADD_CASE_NOTE
+            )
         ),
         entry(
             PAST_CLAIM_DISMISSED_DEADLINE_AWAITING_CAMUNDA.fullName(),
-            List.of(DISMISS_CLAIM)
+            List.of(
+                DISMISS_CLAIM,
+                ADD_CASE_NOTE
```

Comment:
I believe in this scenario because we are merely waiting for Camunda to do its job and move the claim to `CLAIM_DISMISSED_PAST_CLAIM_DISMISSED_DEADLINE` it is already too late. The case should be considered offline and no new notes should be allowed?

Reply:
I'll be honest I don't have the context for this story. Just going off Doug Rice's QA notes here:



CLAIM_DISMISSED_PAST_CLAIM_NOTIFICATION_DEADLINE | Yes |   | Unable to create note.  Expected to see event to add note
-- | -- | -- | --
CLAIM_DISMISSED_PAST_CLAIM_DETAILS_NOTIFICATION_DEADLINE | Yes |   | Unable to create note.  Expected to see event to add note

Reply:
Yes both these scenarios will pass now with your fix above allowing `PAST_CLAIM_NOTIFICATION_DEADLINE_AWAITING_CAMUNDA` and `PAST_CLAIM_DETAILS_NOTIFICATION_DEADLINE_AWAITING_CAMUNDA` flow states to trigger an add note event. These are the only two scenarios that were missing. 

`PAST_CLAIM_DISMISSED_DEADLINE_AWAITING_CAMUNDA` has nothing to do with the above :-)